Build fails in clang i386

Getting these errors in version 2.1.2 in the FreeBSD port on FreeBSD 12 i386:
```
src/_image_wrapper.cpp:386:24: error: non-constant-expression cannot be narrowed from type 'unsigned int' to 'npy_intp' (aka 'int') in initializer list [-Wc++11-narrowing]
    npy_intp dim[3] = {rows, cols, 4};
                       ^~~~
src/_image_wrapper.cpp:386:24: note: insert an explicit cast to silence this issue
    npy_intp dim[3] = {rows, cols, 4};
                       ^~~~
                       static_cast<npy_intp>( )
src/_image_wrapper.cpp:386:30: error: non-constant-expression cannot be narrowed from type 'unsigned int' to 'npy_intp' (aka 'int') in initializer list [-Wc++11-narrowing]
    npy_intp dim[3] = {rows, cols, 4};
                             ^~~~
src/_image_wrapper.cpp:386:30: note: insert an explicit cast to silence this issue
    npy_intp dim[3] = {rows, cols, 4};
                             ^~~~
                             static_cast<npy_intp>( )
src/_image_wrapper.cpp:431:24: error: non-constant-expression cannot be narrowed from type 'unsigned int' to 'npy_intp' (aka 'int') in initializer list [-Wc++11-narrowing]
    npy_intp dim[3] = {rows, cols, 4};
                       ^~~~
src/_image_wrapper.cpp:431:24: note: insert an explicit cast to silence this issue
    npy_intp dim[3] = {rows, cols, 4};
                       ^~~~
                       static_cast<npy_intp>( )
src/_image_wrapper.cpp:431:30: error: non-constant-expression cannot be narrowed from type 'unsigned int' to 'npy_intp' (aka 'int') in initializer list [-Wc++11-narrowing]
    npy_intp dim[3] = {rows, cols, 4};
                             ^~~~
src/_image_wrapper.cpp:431:30: note: insert an explicit cast to silence this issue
    npy_intp dim[3] = {rows, cols, 4};
                             ^~~~
                             static_cast<npy_intp>( )
                                                                                                                                                           1574,5        99%
```

--- a/src/_image_wrapper.cpp
+++ b/src/_image_wrapper.cpp
@@ -352,13 +352,12 @@ static PyObject *image_pcolor(PyObject *self, PyObject *args, PyObject *kwds)
     numpy::array_view<const float, 1> x;
     numpy::array_view<const float, 1> y;
     numpy::array_view<const agg::int8u, 3> d;
-    unsigned int rows;
-    unsigned int cols;
+    npy_intp rows, cols;
     float bounds[4];
     int interpolation;
 
     if (!PyArg_ParseTuple(args,
-                          "O&O&O&II(ffff)i:pcolor",
+                          "O&O&O&nn(ffff)i:pcolor",
                           &x.converter,
                           &x,
                           &y.converter,
@@ -396,13 +395,12 @@ static PyObject *image_pcolor2(PyObject *self, PyObject *args, PyObject *kwds)
     numpy::array_view<const double, 1> x;
     numpy::array_view<const double, 1> y;
     numpy::array_view<const agg::int8u, 3> d;
-    unsigned int rows;
-    unsigned int cols;
+    npy_intp rows, cols;
     float bounds[4];
     numpy::array_view<const agg::int8u, 1> bg;
 
     if (!PyArg_ParseTuple(args,
-                          "O&O&O&II(ffff)O&:pcolor2",
+                          "O&O&O&nn(ffff)O&:pcolor2",
                           &x.converter_contiguous,
                           &x,
                           &y.converter_contiguous,

